spec/requests/home_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe 'HomeController', type: :request do
+  describe 'GET :index' do
+    it 'is successful' do
+      get root_path
+      expect(root_path).to eq('/en')
+
+      expect(response.body).to include('The calculator tells you how many')
+      expect(response).to render_template(:index)
+      expect(response).to be_successful
+    end
+
+    context 'with default locale' do
+      it 'is successful' do
+        get root_path
```

Comment:
нашо його все на кучу ліпить)
```suggestion
        get root_path

```